src/FSharp.UMX.fs

```diff
@@ -33,6 +33,7 @@ type UMX =
     static member inline tag<[<Measure>]'m> (x : int64) : int64<'m> = Unsafe.cast x
     static member inline tag<[<Measure>]'m> (x : uint64) : uint64<'m> = Unsafe.cast x
     static member inline tag<[<Measure>]'m> (x : float) : float<'m> = Unsafe.cast x
+    static member inline tag<[<Measure>]'m> (x : float32) : float32<'m> = Unsafe.cast x
```

Comment:
Minor nit-pick: order lower precision types above higher precision types.
```suggestion
    static member inline tag<[<Measure>]'m> (x : float32) : float32<'m> = Unsafe.cast x
    static member inline tag<[<Measure>]'m> (x : float) : float<'m> = Unsafe.cast x
```

Reply:
I didn't even think about that. Thank for the nit-pick.